tests/integration/mutation/create/with_default_values_test.go

```diff
@@ -0,0 +1,137 @@
+// Copyright 2022 Democratized Data Foundation
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package create
+
+import (
+	"testing"
+
+	testUtils "github.com/sourcenetwork/defradb/tests/integration"
+)
+
+func TestMutationCreate_WithDefaultValues_NoValuesProvided_SetsDefaultValue(t *testing.T) {
+	test := testUtils.TestCase{
+		Description: "Simple create mutation, with default values and no values provided",
+		Actions: []any{
+			testUtils.SchemaUpdate{
+				Schema: `
+					type Users {
+						age: Int @default(int: 40)
+						active: Boolean @default(bool: true)
+						name: String @default(string: "Bob")
+						points: Float @default(float: 10)
+					}
+				`,
+			},
+			testUtils.Request{
+				Request: `mutation {
+							create_Users(input: {}) {
+								age
+								active
+								name
+								points
+							}
+						}`,
```

Comment:
question/todo: Perhaps document and add another test where the same mutation is requested twice when a unique constraint is there and when duplicate is allowed.

Reply:
done